docs/guidance/model/milestones.md

```diff
@@ -0,0 +1,52 @@
+# Milestones
+
+Milestones can be included within the planning, tender, contract and contract implementation sections. 
+
+* The planning milestone block may describe anticipated milestones during the planning stage, such as the preparation 
+of key studies. Usually using the 'preProcurement' milestone type code.
+
+* The tender milestone block should be used to describe anticipated milestones during the proposed delivery 
+of a contract, as well as key dates from the process of tendering and awarding the contract.
+
+* The contract milestone block should be used for events related to the signing of the contract documents.
+
+* The contract implementation milestone block should be used for any event related to delivery of the contract.
+
+The nature of the milestone is indicated by the [milestone type codelist](../../schema/codelists#milestone-type).
+
+The lifecycle of the milestone should be modeled with the fields: `dueDate`, `dateMet`, `dateModified` and [status](../../schema/codelists/#milestone-status).
+
+## Worked example
+
+In the example below:
+
+* A contract is awarded to a user committee with milestones for project commencement and project completion.
+* The project starts later than expected. The project commencement milestone is updated with actual start date and the project completion milestone `status` is set to `notMet`.
```

Comment:
"with actual start date" -> "with the actual start date" (to be consistent with the example below.